### ISSUE TITLE
Invalid Data Dource Error when loading ply file

### ISSUE THREAD

**Issue by jbp35** [qgis/QGIS#56151]
### What is the bug or the crash?

When trying to add a .ply file to a Qgis project, I get an invalid Data Dource Error when the file is located in the following path:
'C:/Users/jlopr/Atelier Pomme/Projets - LTR_Faisabilité Economique/04_Dossier de travail/Qgis/data/demtomesh/mesh_project_area/mesh_project_area_georef.ply'

I don't get the error when moving the same file to the following folder:
'C:\dev\mesh_project_area_georef.ply'

Error message:
Invalid Data Source: C:/Users/jlopr/Atelier Pomme/Projets - LTR_Faisabilité Economique/04_Dossier de travail/Qgis/data/demtomesh/mesh_project_area/mesh_project_area_georef.ply is not a valid or recognized data source.

### Steps to reproduce the issue

drag and drop a file

### Versions

QGIS version
3.34.2-Prizren
QGIS code revision
7d199797fc
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.8.2
PROJ version
9.3.1
EPSG Registry database version
v10.098 (2023-11-24)
GEOS version
3.12.1-CAPI-1.18.1
SQLite version
3.41.1
PDAL version
2.6.0
PostgreSQL client version
15.2
SpatiaLite version
5.1.0
QWT version
6.1.6
QScintilla2 version
2.13.4
OS version
Windows 10 Version 2009

Active Python plugins
layout_panel-main
0.3
quick_map_services
0.19.34
valuetool
3.0.17
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by agiudiceandrea** [comment]
The issue is due to the presence of non ASCII characters in the file path/name.

**Comment by jbp35** [comment]
yes most likely - just wanted to report in case this can fixed easily
